twake/frontend/.storybook/main.js

```diff
@@ -66,4 +66,5 @@ module.exports = {
   core: {
     builder: 'webpack5',
   },
+  staticDirs: ['../public'],
```

Comment:
Why did you add this ? (it produce a bug in the tests)

Reply:
i removed it from the config file and added it when actually running storybook ( in package.json )

Reply:
In this case change the package.json with:
```
    "build-storybook": "build-storybook -o tmp_storybook; rm -rf public/storybook; mv tmp_storybook public/storybook",
```

And also add to the git ignore `tmp_storybook`

The tests will pass in this case